src/legacy/ui/public/vis/editors/default/_agg_select.scss

```diff
@@ -20,3 +20,7 @@
 .visEditorAggSelect__helpLink {
   @include euiFontSizeXS;
 }
+
+vis-agg-select-react-wrapper .euiFormRow {
```

Comment:
@cchaos These styles will be removed as soon as `agg-params` directive is migrated.